scripts/accumulator-server.py

```diff
@@ -81,42 +81,57 @@ def usage():
     Print usage message
     """
 
-    print 'Usage: %s --host <host> --port <port> --url <server url> --pretty-print -v -u' % os.path.basename(__file__)
-    print ''
-    print 'Parameters:'
-    print "  --host <host>: host to use database to use (default is '0.0.0.0')"
-    print "  --port <port>: port to use (default is 1028)"
-    print "  --mqttHost <host>: mqtt broker host to use (if not defined MQTT listening is not enabled)"
-    print "  --mqttPort <port>: mqtt broker port to use (default is 1883)"
-    print "  --mqttTopic <topic>: mqtt topic to use (default is accumulate_mqtt)"
-    print "  --url <server url>: server URL to use (default is /accumulate)"
-    print "  --pretty-print: pretty print mode"
-    print "  --https: start in https"
-    print "  --key: key file (only used if https is enabled)"
-    print "  --cert: cert file (only used if https is enabled)"
-    print "  -v: verbose mode"
-    print "  -u: print this usage message"
+    print(f"Usage: {os.path.basename(__file__)} --host <host> --port <port> --url <server url> --pretty-print -v -u")
+    print("""\nParameters:
+          --host <host>: host to use database to use (default is '0.0.0.0')"
+          --port <port>: port to use (default is 1028)"
+          --mqttHost <host>: mqtt broker host to use (if not defined MQTT listening is not enabled)"
+          --mqttPort <port>: mqtt broker port to use (default is 1883)"
+          --mqttTopic <topic>: mqtt topic to use (default is accumulate_mqtt)"
+          --url <server url>: server URL to use (default is /accumulate)"
+          --pretty-print: pretty print mode"
+          --https: start in https"
+          --key: key file (only used if https is enabled)"
+          --cert: cert file (only used if https is enabled)"
```

Comment:
```suggestion
          --key <key_file>: key file (only used if https is enabled)"
          --cert <cert_file>: cert file (only used if https is enabled)"
```

This is not actually related with the conversion to Python 3 (i.e. it was already wrong in the original script) but we can take this opportunity to fix it :)